### PR TITLE
fix: allow chromicons v2 as a peerDependency

### PR DESCRIPTION
**Pull request by smolster**
## Motivation
See [this internal Slack thread](https://lifeomic.slack.com/archives/CHMFL4PQT/p1636471415066500).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,14 +30,14 @@
     }
   },
   "peerDependencies": {
-    "@lifeomic/chromicons": "^1.2.1",
+    "@lifeomic/chromicons": ">=1.2.1",
```
Comment (sternetj) [inline package.json:33]: Should this one be bumped higher as well?

File: package.json [hunk 2]
```diff
@@ -30,14 +30,14 @@
...
     "react": "16",
     "react-dom": "16",
     "react-router-dom": ">=5.1.2"
   },
   "devDependencies": {
     "@babel/core": "^7.0.0-0",
-    "@lifeomic/chromicons": "^1.2.1",
+    "@lifeomic/chromicons": "^2.2.0",
```
Comment (sternetj) [inline package.json:40]: Pr #222 and we're updating it to version 2.2 🤩

**Review by xdumaine — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (smolster) [reply package.json:33]: If we bump this to e.g >=2.2.0, it will unnecessarily cause warnings for consumers still using v1

Comment (sternetj) [reply package.json:33]: Ahh okay that makes sense

**Review by sternetj — APPROVED** [review]

**Review by HaleyWardo — APPROVED** [review]

**Pull request merged**